backup.js

```diff
@@ -208,6 +212,7 @@ const exportCollection = async ({
 		...CONFIG.MONGO.CREDENTIALS_ARGS,
 		'--collection', collection,
 		'--out', filePath,
+		args['--sort'] ? `--sort "${args['--sort']}"`: undefined,
```

Comment:
Codacy found an issue: [Operator ':' must be spaced.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6060360)